plugins/github.js

```diff
@@ -10,24 +10,35 @@ exports.init = function(config, cimpler) {
          return next();
       }
 
-      try {
-         var build = extractBuildInfo(req.body);
-         if (build) {
+      const payload = JSON.parse(req.body.payload);
```

Comment:
You lost the `try` around this section. If the `payload` doesn't have a `ref` property, it could crash on a ReferenceError at https://github.com/iFixit/ifixit/blob/master/Conf/httpd/a_ifixit.conf#L40, right?

Reply:
Good point.  I can make that change.